x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvContainsTests.java

```diff
@@ -0,0 +1,368 @@
+                    List<TestCaseSupplier.TypedData> typeDataWithNull = new ArrayList<>(originalTestCase.getData());
+                    var data = typeDataWithNull.get(nullPosition);
+                    typeDataWithNull.set(nullPosition, data.withData(data.isMultiRow() ? Collections.singletonList(null) : null));
+                    TestCaseSupplier.TypedData nulledData = originalTestCase.getData().get(nullPosition);
+                    return new TestCaseSupplier.TestCase(
+                        typeDataWithNull,
+                        evaluatorToString.evaluatorToString(nullPosition, nulledData, originalTestCase.evaluatorToString()),
+                        expectedType.expectedType(nullPosition, DataType.BOOLEAN, originalTestCase),
+                        equalTo(nullPosition == 1)
+                    );
+                }));
+
+                if (firstTimeSeenSignature) {
+                    var typesWithNull = new ArrayList<>(original.types());
+                    typesWithNull.set(nullPosition, DataType.NULL);
+                    boolean newSignature = uniqueSignatures.add(typesWithNull);
+                    if (newSignature) {
+                        suppliers.add(
+                            new TestCaseSupplier(
+                                "G2: " + toSpaceSeparatedString(typesWithNull) + " null in " + nullPosition,
+                                typesWithNull,
+                                () -> {
+                                    TestCaseSupplier.TestCase originalTestCase = original.get();
+                                    var typeDataWithNull = new ArrayList<>(originalTestCase.getData());
+                                    typeDataWithNull.set(
+                                        nullPosition,
+                                        typeDataWithNull.get(nullPosition).isMultiRow() ? MULTI_ROW_NULL : NULL
+                                    );
+                                    return new TestCaseSupplier.TestCase(
+                                        typeDataWithNull,
+                                        nullPosition == 0 ? "IsNullEvaluator[field=Attribute[channel=1]]" : "ConstantTrue",
+                                        expectedType.expectedType(nullPosition, DataType.BOOLEAN, originalTestCase),
+                                        equalTo(nullPosition == 1)
+                                    );
+                                }
+                            )
+                        );
+                    }
+                }
+            }
+        }
+
+        return suppliers;
+    }
+
+    private static String toSpaceSeparatedString(ArrayList<DataType> typesWithNull) {
+        return typesWithNull.stream().map(Objects::toString).collect(Collectors.joining(" "));
+    }
+
+    // We always return a boolean.
+    @Override
+    protected Matcher<Object> allNullsMatcher() {
+        return anyOf(equalTo(false), equalTo(true));
```

Comment:
Checking the `allNullsMatcher()` implementation and callers, I see this is for when all of the row values are null. I wonder if it failed because of the old null evaluator error. But I guess this should work with just:
```suggestion
        return equalTo(true);
```

Reply:
yeah while the position is random for the page position which is a row and tests indeed pass. Addressed in https://github.com/elastic/elasticsearch/pull/133636/commits/d33e3332bafacbfa114a86b587f57e9a6ab9b6ad